### PR TITLE
Inline component documentation

### PR DESCRIPTION
**Pull request by marcialfps**

### CODE REVIEW

File: website/screens/components/inline/code/InlineCodePage.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+const sections = [
+  {
+    title: "Props",
+    content: (
+      <DxcTable>
+        <tr>
+          <th>Name</th>
+          <th>Default</th>
+          <th>Description</th>
```
Comment (Lisa159) [inline website/screens/components/inline/code/InlineCodePage.tsx:18]: Change `th` from Description to `HeaderDescriptionCell`.

Comment (marcialfps) [reply website/screens/components/inline/code/InlineCodePage.tsx]: Done!

File: website/screens/components/inline/code/examples/basicUsage.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { DxcInline } from "@dxc-technology/halstack-react";
+import Placeholder from "./Placeholder";
+
+const code = `() => {
+  return (
+    <DxcInline gutter="0.5rem" divider>
```
Comment (Lisa159) [inline website/screens/components/inline/code/examples/basicUsage.ts:6]: What do you think about adding DxcInset to wrap the example?

From this:
image

To this:
image

Comment (marcialfps) [reply website/screens/components/inline/code/examples/basicUsage.ts]: Yes, it is a good idea!

**Review by Lisa159 — APPROVED** [review]

**Pull request merged**